src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -92,10 +92,13 @@ public static RequestNoticeSender using(Clients clients) {
   private final EventPublisher eventPublisher;
   protected final LocationRepository locationRepository;
 
+  private Long recallRequestSize = 0l;
```

Comment:
can it be just long?

Reply:
I mostly changing this to a boolean, my next committ will reflect it ..